lib/msal-common/src/authority/Authority.ts

```diff
@@ -438,33 +443,48 @@ export class Authority {
      * @param newMetadata
      */
     private async updateCloudDiscoveryMetadata(metadataEntity: AuthorityMetadataEntity): Promise<AuthorityMetadataSource> {
+        this.logger.verbose("Attempting to get cloud discovery metadata in the config");
         let metadata = this.getCloudDiscoveryMetadataFromConfig();
         if (metadata) {
+            this.logger.verbose("Found cloud discovery metata in the config.");
```

Comment:
```suggestion
            this.logger.verbose("Found cloud discovery metadata in the config.");
```
Similar typo in few places below.